eftest/src/eftest/runner.clj

```diff
@@ -120,18 +120,22 @@
       @test/*report-counters*)))
 
 (defn- test-all [vars {:as opts
-                       :keys [capture-output? executor]
+                       :keys [capture-output?]
                        :or {capture-output? true}}]
-  (let [report (synchronize test/report)
-        mapf   (if (multithread-namespaces? opts)
-                 (partial pmap* executor)
-                 map)
-        f      #(->> (group-by (comp :ns meta) vars)
-                     (mapf (fn [[ns vars]] (test-ns ns vars report opts)))
-                     (apply merge-with +))]
-    (if capture-output?
-      (capture/with-capture (f))
-      (f))))
+  (let [report             (synchronize test/report)
+        namespace-executor (when (multithread-namespaces? opts)
```

Comment:
Change the name to `executor`. Too much whitespace makes it harder to read the other bindings in the `let`.

Also, rather than using `when`, a `delay` might be a better option here, e.g.

```
(delay (Executors/newCachedThreadPool))
```

Reply:
ok, this is all done